ArchiSteamFarm/Actions.cs

```diff
@@ -282,9 +282,16 @@ public sealed class Actions : IDisposable {
 					TradingScheduled = false;
 				}
 
-				HashSet<Steam.Asset> inventory = await Bot.ArchiWebHandler.GetInventory(Bot.SteamID, appID, contextID, tradable: true, wantedRealAppIDs: wantedRealAppIDs, unwantedRealAppIDs: unwantedRealAppIDs, wantedTypes: wantedTypes).ConfigureAwait(false);
+				HashSet<Steam.Asset> inventory;
 
-				if ((inventory == null) || (inventory.Count == 0)) {
+				try {
+					inventory = await Bot.ArchiWebHandler.GetInventoryEnumerable(Bot.SteamID, appID, contextID).Where(item => item.Tradable && (wantedRealAppIDs?.Contains(item.RealAppID) != false) && (unwantedRealAppIDs?.Contains(item.RealAppID) != true) && (wantedTypes?.Contains(item.Type) != false)).ToHashSetAsync().ConfigureAwait(false);
+				} catch (Exception e) {
+					// Returning exception message instead of logging it
+					return (false, string.Format(Strings.WarningFailedWithError, e.Message));
```

Comment:
You should do that only with `IOException`, normal exception should be logged as well.